CODE_OF_CONDUCT.md

```diff
@@ -14,10 +14,10 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
 include:
 
-- Using welcoming and inclusive language
+- Using welcoming and [inclusive language](https://developers.google.com/style/inclusive-documentation)
 - Being respectful of differing viewpoints and experiences
 - Gracefully accepting constructive criticism
-- Focusing on what is best for the community
+- Focusing on the bigger picture: what is best for the community?
```

Comment:
not sure if this change adds much, I'd recommend removing.

(for context: On governance docs I'm a big nitpicker/stickler on small wording changes that are more for style than substance. In this case, this line is actually taken from a well-regarded CoC template that I wouldn't want to deviate from unless we had a good reason/goal in mind.)